src/components/Profile/APIcall.js

```diff
@@ -36,34 +31,43 @@ function APIcall() {
 
   const onFailure = (response) => console.error(response)
 
-  const TokenContext = React.createContext(localStorage.getItem('ACCESS TOKEN'))
+  const TokenContext = React.createContext()
+
+  let TokenSave = localStorage.getItem('ACCESS TOKEN')
 
   return (
-    <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
-      <Button mode='normal'>
-        {localStorage.getItem('ACCESS TOKEN') ? (
-          <GitHubLogin
-            clientId='69bc88033c4b1bc2b4dc'
-            redirectUri='http://localhost:3000/'
-            onSuccess={onSuccess_disconnect}
-            onFailure={onFailure_disconnect}
-            buttonText={boolButtonText_disconnect}
-            // buttonText='DISCONNECT'
-            className='buttonCustomStyle'
-          />
-        ) : (
-          <GitHubLogin
-            clientId='69bc88033c4b1bc2b4dc'
-            redirectUri='http://localhost:3000/'
-            onSuccess={onSuccess}
-            onFailure={onFailure}
-            buttonText={boolButtonText}
-            //buttonText='CONNECT'
-            className='buttonCustomStyle'
-          />
-        )}
-      </Button>
-    </div>
+    (
+      <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
+        <Button mode='normal'>
+          {localStorage.getItem('ACCESS TOKEN') ? (
+            <GitHubLogin
+              clientId='69bc88033c4b1bc2b4dc'
+              redirectUri='http://localhost:3000/'
+              onSuccess={onSuccess_disconnect}
+              onFailure={onFailure_disconnect}
+              buttonText={boolButtonText_disconnect}
+              className='buttonCustomStyle'
+            />
+          ) : (
+            <GitHubLogin
+              clientId='69bc88033c4b1bc2b4dc'
+              redirectUri='http://localhost:3000/'
+              onSuccess={onSuccess}
+              onFailure={onFailure}
+              buttonText={boolButtonText}
+              className='buttonCustomStyle'
+            />
+          )}
+        </Button>
+      </div>
+    ),
+    (
+      <TokenContext.Provider
```

Comment:
This Context code block is hiding the `CONNECT` `DISCONNECT` button for some reason.

Reply:
It was for testing purpose only, will remove it in the next commit